analysis/wwz/get_wwz_yields.py

```diff
@@ -740,22 +741,22 @@ def make_plots(histo_dict,grouping_mc,grouping_data,save_dir_path,apply_nsf_to_c
 
             ######
             # Print stuff if you want to
-            #if (cat_name == "cr_4l_sf" or cat_name == "cr_4l_btag_of" or cat_name=="cr_4l_btag_sf_offZ_met80") and var_name == "nleps":
-                #print(f"\n{cat_name} {var_name}:")
-                #print("Yields")
-                #data = sum(sum(histo_grouped_data.values(flow=True)))
-                #data_error = sum(sum(histo_grouped_data.variances(flow=True))) ** 0.5
-                #mc = sum(sum(histo_grouped_mc.values(flow=True)))
-                #mc_error = sum(sum(histo_grouped_mc.variances(flow=True))) ** 0.5
-                #data_over_mc_ratio = data / mc
-                #data_over_mc_ratio_error = data_over_mc_ratio * ((data_error / data)**2 + (mc_error / mc)**2)**0.5
-                #print("mc:",mc)
-                #print("mc Error:",mc_error)
-                #print("data:",data)
-                #print("data Error:",data_error)
-                #print("data/mc:", data_over_mc_ratio)
-                #print("data/mc Error:", data_over_mc_ratio_error)
-            #continue
+            if (cat_name == "cr_4l_sf" or cat_name == "cr_4l_btag_of" or cat_name=="cr_4l_btag_sf_offZ_met80") and var_name == "nleps":
+                print(f"\n{cat_name} {var_name}:")
+                print("Yields")
+                data = sum(sum(histo_grouped_data.values(flow=True)))
+                data_error = sum(sum(histo_grouped_data.variances(flow=True))) ** 0.5
+                mc = sum(sum(histo_grouped_mc.values(flow=True)))
+                mc_error = sum(sum(histo_grouped_mc.variances(flow=True))) ** 0.5
+                data_over_mc_ratio = data / mc
+                data_over_mc_ratio_error = data_over_mc_ratio * ((data_error / data)**2 + (mc_error / mc)**2)**0.5
+                print("mc:",mc)
+                print("mc Error:",mc_error)
+                print("data:",data)
+                print("data Error:",data_error)
+                print("data/mc:", data_over_mc_ratio)
+                print("data/mc Error:", data_over_mc_ratio_error)
+            continue
```

Comment:
Don't think we want the continue in by default. Might be clearer just to restore this block to the current commented version. 

Again eventually probably nice to add an option for it. But don't need it for now.